libnmstate/nm/active_connection.py

```diff
@@ -20,6 +20,11 @@
 import logging
 
 from . import nmclient
+from .nmclient import GLib
+from .nmclient import NM
+
+
+NM_MANAGER_ERROR_DOMAIN = GLib.quark_to_string(NM.ManagerError.quark())
```

Comment:
I prefer you do this in the mainloop object which is a singletone. Doing this here will attempt to do this on module import, which may be bad for unit tests (depending on a 3rd party which may or may not be available).